build/DownloadCFamilyPlugin/DownloadCFamilyPlugin.csproj

```diff
@@ -1,5 +1,32 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <!-- This target exists to help test the code in this project.
```

Comment:
New target to help with testing/debugging the build tooling. It won't be executed unless you call it directly, so it won't have any impact on the "normal" build.

Reply:
I'm not sure I understand what exactly is the purpose of this target, how does it help? Does it just invoke the build with hard-coded "dummy" values (folder + jar path?)?

The name Test is a bit confusing, as it sounds like something related to unit tests. Same goes for the parameters, e.g. TestTargetDirectory -- that definitely sounds like something for unit tests. Perhaps something like "DummyParams" ?

Reply:
It makes it simple to debug this targets file and the tasks in it. Yes, it just builds the project and invokes the target the same way it's called from the `SLVS.csproj`.

If the caller doesn't supply specific values for the two required properties, the defaults will be used.
They're not dummy values; the specified jar will be download, unpacked and dumped to the specified output folder.

The target name was originally something longer, but I changed it to "test" to make it easier to remember and shorter to type. I'll change it to `ManualTest`.